README.md

```diff
@@ -34,7 +34,7 @@ Adding the Ninjachat tag to your website once you're logged in to your account w
 
 4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-5. Find the API key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
+5. Find the API key for the department in the share script side bar, copy and place the key in the field provided in the Tag Configuration.
```

Comment:
Do we have GTM Instructions similar to PushNinja in the app

Reply:
Yes Sir.

Reply:
Can you show me the share script side bar

Reply:
@subramaniyamkolluri Sir
 share script sidebar in app: https://snipboard.io/IOnfq2.jpg
 Google tag manager:https://snipboard.io/ot6CfR.jpg